tools/codeformat.py

```diff
@@ -161,6 +168,16 @@ def main():
     # Expand the globs passed on the command line, or use the default globs above.
     files = []
     if args.files:
+        if args.f:
+            all_files = list_files(PATHS, EXCLUSIONS, TOP)
+            filtered = [
+                f for f in args.files if os.path.normpath(os.path.join(TOP, f)) in all_files
+            ]
+            for r in args.files:
+                if args.v and r not in filtered:
```

Comment:
Makes a bit more sense to never enter this loop if not needed, i.e. `args.v` is loop-invariant so use
```
if args.v:
   for ....
```

Reply:
Good idea, at least for readability (the list of files is not long enough for it to matter for performance). Changed.